bench/db_bench.ml

```diff
@@ -327,13 +345,13 @@ let run input =
   init ();
   Log.app (fun l -> l "\n");
   Log.app (fun l -> l "Fill in random order");
-  let rw = Index.write_random () in
+  let rw, name = Index.write_random () in
   let bench_list =
     [
       ( (fun () -> Index.read_random rw),
         [ `Read `RW; `All; `Minimal; `Index ],
         "RW Read in random order" );
-      ( (fun () -> Index.ro_read_random rw),
+      ( (fun () -> Index.ro_read_random rw name),
```

Comment:
If this benchmark needs a particular index instance `name`, could we just pass it directly here rather than sharing it's path and reconstructing?